docs/source/deployment/azure-functions.md

```diff
@@ -112,6 +112,8 @@ It is important to set output binding name to **$return** to work correctly at t
 }
 ```
 
+**Note:** To enable the health checks it's necessary to [modify the route template](../../../packages/apollo-server-azure-functions/README.md#enabling-health-checks) in the `function.json` file.
```

Comment:
@glasser it seems this broke the docs build, do you know how can I link it or should I just replicate the documentation here?

Reply:
We don't put anything outside of `docs` in the docs site, so I'd recommend just linking to the README via a GitHub main branch or npmjs.com link. Replicating sounds good too though.

I'm hoping that as part of the AS3 push we can improve the integration/deployment docs to be more complete and authoritative.

Reply:
@glasser done, all the checks are passing now!